docs/kyma/14-01-overview.md

```diff
@@ -0,0 +1,8 @@
+---
+title: Overview
+type: Troubleshooting
+---
+
+The troubleshooting section aims to identify the most common recurring problems the users face when they install and start using Kyma, as well as the most suitable solutions to these problems.
+
+If you can't find a solution, don't hesitate to create a [GitHub](https://github.com/kyma-project/kyma/issues) issue or reach out to either the ["installation"](https://kyma-community.slack.com/messages/CD2HJ0E78) or ["general"](https://kyma-community.slack.com/messages/CBLBESMST) Slack channel to get direct support from the community.
```

Comment:
Reading it again, it could be more straightforward just to say" If you can't find a solution, create ..." but if you want to encourage our users to do it then let it be.